src/devices/Common/Interop/Unix/Libasound/Interop.libasound.cs

```diff
@@ -0,0 +1,152 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// Disable these StyleCop rules for this file, as we are using native names here.
+#pragma warning disable SA1300 // Element should begin with upper-case letter
+#pragma warning disable SA1307 // Field should begin with upper-case letter
+#pragma warning disable CS1591
+
+using System;
+using System.Runtime.InteropServices;
+
+#if BUILDING_IOT_DEVICE_BINDINGS
+internal
+#else
+public
+#endif
+    partial class Interop
```

Comment:
I think this will be more readable:

```suggestion
#if BUILDING_IOT_DEVICE_BINDINGS
internal partial class Interop
#else
public partial class Interop
#endif
```

Reply:
same comment for other places